webapp/src/ts/app.component.ts

```diff
@@ -5,26 +5,26 @@ import { Store } from '@ngrx/store';
 import { TranslateService } from '@ngx-translate/core';
 import { setTheme as setBootstrapTheme} from 'ngx-bootstrap/utils';
 import { combineLatest } from 'rxjs';
-
-import { DBSyncService } from './services/db-sync.service'
+import { DBSyncService } from '@mm-services/db-sync.service'
```

Comment:
We started separating the vendor dependencies to our local dependencies by an empty space, and we keep the vendor dependencies in the top section. 
So let's please keep this empty space.